tests/test_train.py

```diff
@@ -132,36 +132,3 @@ def test_config_file(tmp_path: Path) -> None:
     )
     args = [sys.executable, "train.py", "config_file=" + str(config_file)]
     subprocess.run(args, check=True)
-
-
-@pytest.mark.parametrize(
-    "task",
-    [
-        "bigearthnet",
-        "byol",
-        "chesapeake_cvpr",
-        "cowc_counting",
-        "cyclone",
-        "landcoverai",
-        "naipchesapeake",
-        "resisc45",
-        "sen12ms",
-        "so2sat",
-        "ucmerced",
-    ],
-)
-def test_tasks(task: str, tmp_path: Path) -> None:
```

Comment:
These tests are now in `tests/trainers/*.py` and will be run on every commit instead of just on releases.